build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -43,13 +43,18 @@
                     ContinueOnError="WarnAndContinue" />
     </Target>
 
+    <PropertyGroup>
+      <UploadNuGetPackagesArchiveToAzure>false</UploadNuGetPackagesArchiveToAzure>
+    </PropertyGroup>
+
     <Target Name="GenerateNuGetPackagesArchive"
             DependsOnTargets="SetupNuGetPackagesArchiveInputsOutputs"
             Inputs="$(IntermediateArchive)"
             Outputs="$(IntermediateArchive)">
-      <PropertyGroup>
-        <UploadNuGetPackagesArchiveToAzure>true</UploadNuGetPackagesArchiveToAzure>
-      </PropertyGroup>
+
+      <CreateProperty Value="true">
```

Comment:
I am not a fan of comments, but in this case, I think one is warranted. Say that you are using a task here so that the property value only gets set when the target runs.